test/onnx/test_operators.py

```diff
@@ -206,6 +206,10 @@ def test_transpose(self):
         x = torch.tensor([[0.0, 1.0], [2.0, 3.0]], requires_grad=True)
         self.assertONNX(lambda x: x.transpose(0, 1).transpose(1, 0), x)
 
+    def test_numpy_T(self):
```

Comment:
Please don't add this. The expect files are a maintenance burden and we shouldn't add any more, especially if we have another way to test a feature (as we do in this case).